docs/UserGuide.md

```diff
@@ -349,12 +367,26 @@ Put alerts here, if any
 <div markdown="block" class="alert alert-info">
 
 **Note:**
-Put notes here
+``find-a`` is case-insensitive and does not require the whole word to match, for example, ``find-a la`` will match
+with "CS2103T Lab"
```

Comment:
I feel that this can be abit clearer. Something like: 

The keyword is case-insensitive and does not require the whole word to match, for example, if you had an assignment with the name `"CS2100 Lab"`, using the command ``find-a la``, `find-a lab`  or `find-a LAB`will match with this assignment